src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -174,16 +174,16 @@ public abstract class TreeOptions : LearnerInputBaseWithGroupId
         /// <summary>
         /// The seed of the random number generator.
         /// </summary>
-        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the random number generator", ShortName = "r1")]
-        public int RngSeed = 123;
+        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the random number generator", ShortName = "r1,RngSeed")]
+        public int RandomSeed = 123;
 
         // this random seed is only for active feature selection
         /// <summary>
         /// The seed of the active feature selection.
         /// </summary>
-        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the active feature selection", ShortName = "r3", Hide = true)]
+        [Argument(ArgumentType.LastOccurenceWins, HelpText = "The seed of the active feature selection", ShortName = "r3,FeatureSelectSeed", Hide = true)]
         [TGUI(NotGui = true)]
-        public int FeatureSelectSeed = 123;
+        public int FeatureSelectionRandomSeed = 123;
```

Comment:
>FeatureSelectionRandomSeed [](start = 19, length = 26)

Do we really need two of them?

Reply:
Kind of. There are two degrees of control that internal people wanted to independently vary or not vary: selection of *features* and selection of *data*. I have not kept touch with this set of people over the years but the reasoning at one point was definitely considered important. Let us not change it.

Reply:
Yes sir.

---
In reply to: [260863640](https://github.com/dotnet/machinelearning/pull/2753#discussion_r260863640) [](ancestors = 260863640)